### PR TITLE
UI Kit | FE | Date / Time Inputs add new isCondensed prop

### PR DESCRIPTION
**Pull request by jmcreasman**
Adding a new boolean isCondensed prop that when set to true condenses the input height, font size and icon size for the following four components:

### DateInput
isCondensed = false
Screenshot 2024-04-22 at 14 02 01

isCondensed = true
Screenshot 2024-04-22 at 14 02 13

### DateRangeInput
isCondensed = false
Screenshot 2024-04-23 at 13 19 07

isCondensed = true
Screenshot 2024-04-23 at 13 19 14

### TimeInput
isCondensed = false
Screenshot 2024-04-23 at 13 19 49

isCondensed = true
Screenshot 2024-04-23 at 13 20 00

### DateTimeInput
isCondensed = false
Screenshot 2024-04-23 at 13 20 26

isCondensed = true
Screenshot 2024-04-23 at 13 20 34

### CODE REVIEW

**Review by ddouglasz — COMMENTED** [review]
Looks good. 
Thank you 🙌🏽

**Review by CarlosCortizasCT — APPROVED** [review]
Thanks @jmcreasman 👍

**Review by chloe0592 — APPROVED** [review]
Thank you!

**Comment by CarlosCortizasCT** [comment]
> @CarlosCortizasCT It looks like some of the other Date/Time related input can all be done in the same PR as the bulk of the work is already done here and they just need to pass in the isCondensed prop. If merging this PR can be held off until I've done that I figure it would be preferable than multiple PRs to be reviewed. Thanks!

Sure! I think that's a great idea 👍

**Comment by jmcreasman** [comment]
Ok, added the other three and this is ready for any additional review again.

File: .changeset/modern-berries-deliver.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@commercetools-uikit/date-input': minor
```
Comment (CarlosCortizasCT) [inline .changeset/modern-berries-deliver.md:2]: Could you please update the changeset to include all the components that have been updated? 🙏

**Review by FilPob — APPROVED** [review]

File: .changeset/modern-berries-deliver.md (hunk 1 above)
Comment (jmcreasman) [reply .changeset/modern-berries-deliver.md:2]: @CarlosCortizasCT sorry about that, edited and pushed the updated changeset now

**Pull request merged**